### ISSUE TITLE
ViLBERT: Pretraining Task-Agnostic Visiolinguistic Representations for Vision-and-Language Tasks

### ISSUE THREAD

**Issue by SerialLain3170** [SerialLain3170/nagato#16]
## Overview

Inspired by BERT's success, authors applied the techniques to multi-modal learning, combination of images and texts. Differences between BERT and ViLBERT are as follows.

<img width="821" alt="Screen Shot 2021-08-03 at 20 00 38" src="https://user-images.githubusercontent.com/32360147/128006239-396d5c9d-a5a5-4580-937c-7b3d84fd5be0.png">

- Co-Attentional Transformer Layer
  - Attention layer conditioned other representations

<img width="703" alt="Screen Shot 2021-08-03 at 20 01 12" src="https://user-images.githubusercontent.com/32360147/128006249-dc2e1725-e80a-428b-92c3-4772d2b4fa2d.png">

- Image Representations
  - Image region features are generated by extracting bounding boxes gained from Faster-RCNN. In order to consider orders of cropped regions, authors embedded spatial encoding into them.

- Training tasks and objectives
  - Masked multi-modal modeling task
        - When it comes to image representations, the model aims to reconstruct the distribution that is obtained from pre-trained Faster-RCNN.
   - Multi-modal alignment task
        - The model aims to predict whether texts and images are aligned.

**Comment by SerialLain3170** [comment]
## Evaluation

- Authors showed that VilBERT achieved SOTA on 5 tasks.
  - Visual Question Answering (VQA)
  - Visual Commonsense Reasoning (VCR)
  - Grounding Referring Expressions
  - Caption-based Image Retrieval
  - Zero-shot Caption-based Image Retrieval

<img width="814" alt="Screen Shot 2021-08-03 at 20 17 15" src="https://user-images.githubusercontent.com/32360147/128006841-d2e9f13c-d8b5-43bd-a80e-f6868c54d97b.png">

**Comment by SerialLain3170** [comment]
## Supplemental Information

[Paper Link](https://arxiv.org/pdf/1908.02265.pdf)

### Authors
Jiasen Lu, Dhruv Batra, Devi Parikh, Stefan Lee

### Organization
Georgia Institute of Technology, Facebook AI Research, Oregon State University

### Conference
NeurIPS 2019
